inception-project/inception
# #4567 - Add support for a generic CAS RDF export format

**What's in the PR**
- Ported RDF reader/writer from DKPro Core
- Added PARAM_IRI_FEATURES so we can represent KB concepts are proper IRIs in the export instead of strings
- Added test using PARAM_IRI_FEATURES

**How to test manually**
* Export a document as RDF
* Check if the exported TTL looks good
* Import it back again

**Automatic testing**

**Documentation**
